CIP-0048/README.md

```diff
@@ -0,0 +1,249 @@
+---
+CIP: 48?
+Title: 721 References and Payloads
+Authors: Jack (ada9000, https://forum.cardano.org/u/jack7e/, @ada9000_ twitter)
+Comments-URI: https://forum.cardano.org/t/cip-extended-721/
+Status: Draft
+Type: Informational
+Created: 2022-04-22 (updated 2022-09-22)
+---
+
+## Important note
+
+This CIP mentions the `ext` json key defined by [CIP49](https://github.com/cardano-foundation/CIPs/pull/343/files) which is an outcome of work on CIP48.
```

Comment:
```suggestion
```
My understanding is that bridging this to CIP88 means that the proposal formerly called CIP49 is now deprecated & I've marked it such on https://github.com/cardano-foundation/CIPs/pull/343 - so if you think otherwise be sure to post in that CIP.  In any case any "important note" should be removed from here and placed on one of the top level document sections.

Duplicate of https://github.com/cardano-foundation/CIPs/pull/249/files#r1149624522.